test/micro/org/openjdk/bench/java/lang/StringDecode.java

```diff
@@ -46,22 +36,22 @@
 @State(Scope.Thread)
 public class StringDecode {
 
-    @Param({"US-ASCII", "ISO-8859-1", "UTF-8", "MS932", "ISO-8859-6", "ISO-2022-KR"})
```

Comment:
What would you think of retaining the previous set of charset names as a comment -- as a suggestion for someone who wants additional coverage?